src/modbus-rtu.c

```diff
@@ -26,6 +26,13 @@
 #include <linux/serial.h>
 #endif
 
+
+#if ESP_IDF
```

Comment:
why here 

```
#if ESP_IDF
```

and not

```
#if defined(ESP_PLATFORM)
```

Reply:
yeah on the naming (`ESP_IDF` vs `ESP_PLATFORM`) we can put what @stephane think it's best!

Reply:
I have a preference for `ESP_PLATFORM`

Reply:
Me too

Reply:
Changed to `ESP_PLATFORM`!